src/math_llama.jl

```diff
@@ -0,0 +1,44 @@
+"""
+    math_llama.jl
+
+    This file contains the math functions used in the Llama model.
+"""
+
+begin
+# Calculate the root mean square norm of a vector. Reference in llama2.c lines 182-195
```

Comment:
Please use proper docstrings (with `"""`, the function name indented by one tab), see https://docs.julialang.org/en/v1/manual/documentation/#Writing-Documentation or code on the current main branch
Add a docstring for every function and struct, so it will be added automatically to the documentation.